docs/change_controlled.rst

```diff
@@ -0,0 +1,78 @@
+For Data Management, these documents have the DocuShare handle "LDM" and they must be approved by the DM CCB using the :ref:`RFC <decision-making-rfc>` process.
+These change-controlled documents can be written using the Word templates defined in `Document-9224`_, but the preference within Data Management is to write change-controlled documents in LaTeX using the ``lsstdoc`` document class provided by the `lsst-texmf`_ package, and to develop these documents using ``git`` repositories within the LSST organization on `GitHub`_.
+
+.. _CCB: https://project.lsst.org/groups/ccb/
+.. _Document-9224: https://ls.st/Document-9224
+.. _lsst-texmf: https://lsst-texmf.lsst.io
+.. _LPM-19: https://ls.st/LPM-19
+.. _LPM-51: https://ls.st/LPM-51
+.. _GitHub: https://github.com/lsst
+.. _LDM-294: https://ls.st/LDM-294
+
+.. note::
+  The definition of which types of documents should be change-controlled is defined in `LDM-294`_.
+
+
+Developer Workflow
+==================
+
+The development of change-controlled documents differs from :ref:`that of code <git-branching>`, in that some documents under development can take a long time to evolve and can be developed under multiple stories (ticket branches), and by multiple authors.
+Repositories associated with change-controlled documents should therefore use two protected branches: ``master`` should correspond solely to an archived instance of the document on DocuShare, and ``draft`` is a development branch that should be used as the document evolves.
+Ticket branches should branch from ``draft`` and should follow standard :ref:`DM branching policies <git-branching>`.
+
+.. warning::
+  Need to understand whether we need to handle two independent edits to an LDM that will be submitted as two distinct RFCs in parallel.
+
+Once a document handle has been issued a repository can be created in the ``lsst`` GitHub organization using a repository name that matches the handle.
+If the document exists in a pre-existing repository, it should be renamed to the handle and, if necessary, moved to the correct organization.
+
+.. warning::
+  It may be necessary to adjust the history of a pre-existing repository if it has been developed using different policies, in particular it is important that ``master`` be adjusted such that it only contains merge commits from approved versions (and the initial placeholder commit).
+
+All documents use a Travis configuration file to build the document and deploy it to the corresponding https://ldm-nnn.lsst.io/ web site.
+Built PDFs of documents should not be committed to the repository.
+They are deployed automatically to the web during development and the final version will also be archived on DocuShare.
+
+During development:
+
+* Use the ``lsstdraft`` class option to make it clear that the document has not been approved.
```

Comment:
Can we auto-check in Travis to make sure that `lsstdraft` never appears in `master`?  (I don't think we can require that it appear in non-`master`, although that would be desirable.)

Reply:
I suppose that Travis could check although by that time you've already merged it to master.